src/components/Dashboard/Dashboard.tsx

```diff
@@ -103,14 +103,13 @@ export const Dashboard: React.FC<Props> = ({rates, date, dataReady}) => {
                 <h2 className="converter__calc__header">
                     {t('ConvertCurrencyCalculatorHeader')}
                 </h2>
-                {!dataReady ?
-                  <Spin/>
-                  : <>
+                 <Spin spinning={!dataReady} delay={10}>
```

Comment:
Is 10ms delay is the lowest value ?

Reply:
Nope, but I think it's unnecessary to display spinner if data will fetch faster than 10ms :D